src/main/java/hudson/plugins/git/util/GitUtils.java

```diff
@@ -260,8 +260,17 @@ public static EnvVars getPollEnvironment(AbstractProject p, FilePath ws, Launche
         // add env contributing actions' values from last build to environment - fixes JENKINS-22009
         addEnvironmentContributingActionsValues(env, b);
 
+        // Just use environment contributors
+
+        try {  // Getting environment may trigger remoting in some cases which can throw errors.
+            for (EnvironmentContributor ec : EnvironmentContributor.all().reverseView())
```

Comment:
Ah, sorry -- I was just trying to remind, since you asked something that had already come up.